pandas/core/format.py

```diff
@@ -23,6 +24,9 @@
 import itertools
 import csv
 
+class FormatWarning(Warning):
+    pass
```

Comment:
Is it needed to make a custom warning class here? Why not just a UserWarning? (or subclass from that) 
(although I don't know if we have some kind of policy about that)